src/module_library/von_caemmerer_c4_assimilation.cpp

```diff
@@ -0,0 +1,159 @@
+
+#include "von_caemmerer_c4_assimilation.h"
+
+double ElectronTransportModel::operator()(double light_photon_flux_density){
+    double x = light_nondimensionalize(light_photon_flux_density);
+    if (std::isinf(x)){
```

Comment:
`std::isinf` is from the `<cmath>` library, so this file should have a `#include <numeric> // for std::isinf` at the top. This may be another case of an `include` in the header file that should be in the `.cpp` file. It would be good to double check all of the `includes` in the header file and move them to the `.cpp` file if they are only used in the `.cpp` file.

Reply:
Why `#include <numeric>` and not `#include <cmath>`  if `std::isinf` is in `cmath`?

Reply:
That was a typo 😂 If you look again you will see the comment has been edited.